Documentation/ApiOverview/ContentElements/ContentElementsWizard.rst

```diff
@@ -2,167 +2,197 @@
+Any of these entries can be omitted. You **should** at
+least define a title.
 
-      mod.wizards {
-          newContentElement.wizardItems {
-              plugins {
-                  elements {
-                      example_registration {
-                          iconIdentifier = example-registration
-                          title = Registration Example
-                          description = Create a registration form
-                          tt_content_defValues {
-                              CType = list
-                              list_type = example_registration
-                          }
-                      }
-                  }
-              }
-          }
-      }
+New content elements are usually added in extensions in file
+:file:`EXT:my_extension/Configuration/Overrides/tt_content.php`.
 
-   You may want to replace :typoscript:`title` and :typoscript:`description`
-   from above, using language files for translation, for example:
+The following groups are available by default:
 
-   .. code-block:: typoscript
+default
+    ..  versionchanged:: 13.0 This group was renamed from group `common`.
 
-      title = LLL:EXT:example/Resources/Private/Language/locallang.xlf:registration_title
-      description = LLL:EXT:example/Resources/Private/Language/locallang.xlf:registration_description
+    Default group for commonly used content elements
+forms
+    Content elements representing forms like a contact form or a login form
+lists
 
-#. Include TSconfig
+menu
+    Menus that can be inserted as content elements like a sitemap or a menu
+    of all subpages.
+plugins
+    Plugins provided by extensions
+special
+    Content elements that are used of special cases
 
-   .. code-block:: typoscript
-      :caption: EXT:example/Configuration/page.tsconfig
+All content element groups are listed in
+:php:`$TCA['tt_content']['columns']['CType']['config']['itemGrops']` you can
+debug them in the TYPO3 backend using the backend module
+:guilabel:`System > Configuration` if :composer:`typo3/cms-lowlevel` is installed
+and you are and administrator.
 
-      @import 'EXT:example/Configuration/TsConfig/Page/Mod/Wizards/NewContentElement.tsconfig'
+Some third party extensions like :composer:`bk2k/bootstrap-package` are altering
+the available groups.
 
-   This always includes the above page TSconfig. It is better practice to make this configurable by
-   :ref:`registering this file as static page TSconfig <t3tsconfig:register-static-page-tsconfig>`.
+.. _content-element-wizard-plain:
 
-   .. note::
-      The usage of :file:`Configuration/page.tsconfig` is only valid in TYPO3
-      v12+. If you want to stay compatible with TYPO3 v11 and v12 have a look
-      into :ref:`t3tsconfig:setting-page-tsconfig`.
+Plain content elements or plugins
+=================================
 
-#. :ref:`Register your icon <icon-registration>`
+You can add a plain content element or non-extbase plugin using method
```

Comment:
In the other documentation a "plain plugin" was the abstract plugin. I'm confused.

```suggestion
You can add a content element or plain plugin (non-extbase) using method
```